subprojects/enterprise-operations/src/main/java/org/gradle/operations/initialization/ProjectIsolationSettingsFinalizedProgressDetails.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.operations.initialization;
+
+/**
+ * Signals whether Project Isolation has been enabled in the build.
+ *
+ * @since 8.3
+ */
+public interface ProjectIsolationSettingsFinalizedProgressDetails {
```

Comment:
🤔 Should it be called `ProjectIsolation`or `IsolatedProjects` ?

In the only docs we have, we call it "Project Isolation" : https://gradle.github.io/configuration-cache/#project_isolation

On the other hand, the CLI option is called "Isolated Projects", and this is how it is called in the rest of the source code as well.

Reply:
After synching with the team, `ProjectIsolation` is the way to go

Reply:
⚙️ I think the code entity should be called `IsolatedProjects` to go with the existing types and make searching for related code easier.